src/components/SectionHeader.tsx

```diff
@@ -60,73 +60,16 @@ function CircleFilter() {
         setMousePosition({ x: e.clientX - rect.left, y: e.clientY - rect.top });
       }}
     >
-      <Circle
-        style={
-          {
-            opacity: showCircle ? 1 : 0,
-            "--x-pos": x.toFixed() + "px",
-            "--y-pos": y.toFixed() + "px",
-          } as CSSProperties
-        }
+      <div
+        className="pointer-events-none absolute aspect-square rounded-full transition-opacity duration-300"
+        style={{
+          height: `${diameter}px`,
+          opacity: showCircle ? 1 : 0,
+          top: (y - radius).toFixed() + "px",
+          left: (x - radius).toFixed() + "px",
+          backdropFilter: "saturate(2500%) brightness(433%) hue-rotate(-296deg)",
```

Comment:
updating the site's colors made the math for this circle filter effect wrong and caused it to show the wrong color.